students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,20 @@
+def number_of_elements_equal_to_the_maximum():
+    inputs = []
+    while True:
+        number = int(input())
+        if number == 0:
+            break
+        inputs.append(number)
+    sorted_inputs = sorted(inputs, reverse=True)
+    max = sorted_inputs[0]
```

Comment:
max is built in function. Be carefull:
>>> max
<built-in function max>
>>> max([1,5,2])
5
>>> max = 20
>>> max([1,5,2])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable
>>>